sale_order_customer_order_number/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Customer Order Number",
+    "version": "12.0.1.0.0",
+    "author": "Quartile Limited",
+    "category": "Sale",
+    "website": "https://www.quartile.co",
+    "summary": "",
+    "license": "LGPL-3",
+    "depends": ["sale", "account"],
```

Comment:
```suggestion
    "depends": ["sale"],
```

Reply:
Since the module also adds `client_order_ref` to invoice line, therefore `account` is needed here

Reply:
@TimLai125 account is a distant dependency of sale.